examples/00-fluent/ahmed_body_workflow.py

```diff
@@ -79,10 +80,18 @@
 # =====================================================================================
 
 workflow = session.workflow
-geometry_filename = examples.download_file(
-    "ahmed_body_20_0degree_boi_half.scdoc",
-    "pyfluent/examples/Ahmed-Body-Simulation",
-)
+
+if platform.system() == "Windows":
+    geometry_filename = examples.download_file(
+        "ahmed_body_20_0degree_boi_half.scdoc",
+        "pyfluent/examples/Ahmed-Body-Simulation",
+    )
+else:
+    geometry_filename = examples.download_file(
+        "ahmed_body_20_0degree_boi_half.scdoc.pmdb",
+        "pyfluent/examples/Ahmed-Body-Simulation",
+    )
+
```

Comment:
@abhishekchitwar We should consider the need to linit duplication. E.g.,
```Python
filenames = {
    "Windows": "ahmed_body_20_0degree_boi_half.scdoc",
    "Other": "ahmed_body_20_0degree_boi_half.scdoc.pmdb",
}

geometry_filename = examples.download_file(
    filenames.get(platform.system(), filenames["Other"]),
    "pyfluent/examples/Ahmed-Body-Simulation",
)
```

Reply:
Sure! Got it!